### PR TITLE
Use markdown paths, instead of url links, for smooth version updates going forward

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

**Comment by depombo** [comment]
Hmm the build passes for me locally, very weird

File: versioned_docs/version-0.0.10/Sample Queries/aws_ec2.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Install the AWS EC2 module
 SELECT * FROM iasql_install('aws_ec2');
 ```
 
-Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](/ami) to get the latest AMI ID for a given AMI image and AWS region.
+Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](../ami) to get the latest AMI ID for a given AMI image and AWS region.
```
Comment (dfellis) [inline versioned_docs/version-0.0.10/Sample Queries/aws_ec2.md:16]:
```suggestion
Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](../ami.md) to get the latest AMI ID for a given AMI image and AWS region.
```

File: versioned_docs/version-0.0.8/Sample Queries/aws_ec2.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Install the AWS EC2 module
 SELECT * FROM iasql_install('aws_ec2');
 ```
 
-Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](/ami) to get the latest AMI ID for a given AMI image and AWS region.
+Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](../ami) to get the latest AMI ID for a given AMI image and AWS region.
```
Comment (dfellis) [inline versioned_docs/version-0.0.8/Sample Queries/aws_ec2.md:16]:
```suggestion
Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](../ami.md) to get the latest AMI ID for a given AMI image and AWS region.
```

File: versioned_docs/version-0.0.8/Sample Queries/aws_ec2.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Install the AWS EC2 module
 SELECT * FROM iasql_install('aws_ec2');
 ```
 
-Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](/ami) to get the latest AMI ID for a given AMI image and AWS region.
+Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](../ami.md) to get the latest AMI ID for a given AMI image and AWS region.
```
Comment (dfellis) [inline versioned_docs/version-0.0.8/Sample Queries/aws_ec2.md:16]:
```suggestion
Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](../How-to\ Guides/ami.md) to get the latest AMI ID for a given AMI image and AWS region.
```

**Pull request merged**